packages/cli/lib/model-discoverer.js

```diff
@@ -0,0 +1,111 @@
+const debug = require('./debug')('model-discoverer');
+const fs = require('fs');
+const path = require('path');
+
+/**
+ * Given a datasource and discovery options,
+ * return a list of objects {table: 'foo', schema: 'bar}
+ */
+function discoverModelNames(ds, options) {
+  return new Promise((resolve, reject) => {
+    function discover() {
+      ds.discoverModelDefinitions(options, (err, schemas) => {
```

Comment:
Discovery APIs support Promises, no need to use callbacks.

```ts
const schemas = await ds.discoverModelDefinitions(options);
// etc.
```

Please rewrite all helpers in this file to use `await` instead of callbacks.

Reply:
Done